src/backend/assets/js/main.js

```diff
@@ -112,8 +112,46 @@ const loadVideoAndSlides = function(div, videoURL, slideURL) {
   }
 };
 
-window.handleClientLoad = handleClientLoad;
-window.initClient = initClient;
-window.handleAuthClick = handleAuthClick;
-window.listUpcomingEvents = listUpcomingEvents;
-window.loadVideoAndSlides = loadVideoAndSlides;
+$(document).ready(function () {
+  let filter;
+  //store room filters
+  let room = {
+    dateFilter:'.date-filter'
+  };
+  //store track filters
+  let track = {
+    dateFilter:'.date-filter'
+  };
+  //store schedule filters
+  let schedule = {
+    dateFilter:'.day-filter'
+  };
+
+  const showNoResult = function(filterType , calendarMode) {
+    filter = eval(filterType);
```

Comment:
Here we can do:

```js
const filter = allFilters[filterType];
```

This avoids using `eval` and is clearer for other developers to understand.